src/edit/actions/ExportAction.js

```diff
@@ -29,7 +29,18 @@ L.ExportAction = L.EditAction.extend({
       L.IconUtil.toggleXlink(this._link, 'get_app', 'spinner');
       L.IconUtil.toggleTitle(this._link, 'Export Images', 'Loading...');
       L.IconUtil.addClassToSvg(this._link, 'loader');
-      edit.startExport();
+console.log(this, edit.options)
+// explicitly send each option:
+      edit.startExport({
+        collection: edit.options.collection,
```

Comment:
Here we can see all the settable options being passed into the export start command; there are a lot, allowing for some really detailed customization of both the Export process management and the UI.